lib/coins.py

```diff
@@ -367,6 +368,54 @@ class BitcoinSegwit(BitcoinMixin, Coin):
         'ELEX01.blackpole.online s t',
     ]
 
+class Emercoin(Coin):
+    NAME = "Emercoin"
+    SHORTNAME = "EMC"
+    NET = "mainnet"
+    XPUB_VERBYTES = bytes.fromhex("0488b21e")
+    XPRV_VERBYTES = bytes.fromhex("0488ade4")
+    P2PKH_VERBYTE = bytes.fromhex("21")
+    P2SH_VERBYTES = [bytes.fromhex("5c")]
+    WIF_BYTE = bytes.fromhex("80")
+    GENESIS_HASH = ('00000000bcccd459d036a588d1008fce'
+                    '8da3754b205736f32ddfd35350e84c2d')
+    TX_COUNT = 217380620
+    TX_COUNT_HEIGHT = 464000
+    TX_PER_BLOCK = 1700
+    VALUE_PER_COIN = 1000000
+    RPC_PORT = 6662
+
+    DESERIALIZER = DeserializerTxTimeAuxPow
+
+    PEERS = []
+
+    @classmethod
+    def sanitize_url(cls, url):
+        # Remove surrounding ws and trailing /s
+        url = url.strip().rstrip('/')
+        match = cls.RPC_URL_REGEX.match(url)
+        if not match:
+            raise CoinError('invalid daemon URL: "{}"'.format(url))
+        if match.groups()[1] is None:
+            url += ':{:d}'.format(cls.RPC_PORT)
+        if not url.startswith('http://') and not url.startswith('https://'):
```

Comment:
I don't think we need a separate function just for this change.  I don't think it would hurt to be in the base class implementation.  Can you do that, then I'll apply this.  Thanks.